### PR TITLE
docs: update and cleanup tutorials and examples

### PR DESCRIPTION
**Pull request by sathvikbhagavan**

### CODE REVIEW

File: docs/src/examples/nonlinear_elliptic.md [hunk 1]
```diff
@@ -79,18 +79,14 @@ input_ = length(domains)
 n = 15
 chain = [Lux.Chain(Dense(input_, n, Lux.σ), Dense(n, n, Lux.σ), Dense(n, 1)) for _ in 1:6] # 1:number of @variables
 
-strategy = QuadratureTraining()
+strategy = GridTraining(0.01)
```
Comment (ChrisRackauckas) [inline docs/src/examples/nonlinear_elliptic.md:82]: We shouldn't default to grid training.

**Comment by ChrisRackauckas** [comment]
Looks like tests fail?

**Comment by sathvikbhagavan** [comment]
Yeah, doc build is timing out. Currently, checking examples which are slow and trying to speed it up.

**Comment by ChrisRackauckas** [comment]
I'm not sure the regular queue would work because some want to use GPUs?

**Comment by sathvikbhagavan** [comment]
>I'm not sure the regular queue would work because some want to use GPUs?

Yes, I was testing out the build times in the normal queue. I have reverted back to GPU queue

**Comment by ChrisRackauckas** [comment]
Do we know which ones take a long time? @mortonpi is it possible to make the documenter debug output spit out timing information?

**Pull request merged**